src/pages/settings/AddLoginPage.js

```diff
@@ -0,0 +1,156 @@
+import React, {Component} from 'react';
```

Comment:
Also -- and let me know if this is done intentionally so as to not mess with some navigation stuff -- but should the path of this file be `src/pages/settings/profile/AddLoginPage.js` since it's kind of a subpage from the Profile page, not the main settings page? 

(also I think we should rename the `ProfilePage` folder to `profile` to match the convention we seem to be using for folders

Reply:
I left it in settings because it doesn't necessarily need to be accessed via profile (in case other spots want to navigate to this page as well) but I can move it in there if it's clearer.

Regarding folder naming convention, it seemed like pages were always Pascal-cased (looking at pages/signin subfolders)

Reply:
Ah yeah. I was thinking of like `pages/home/sidebar` but I see even within that we have `pages/home/sidebar/OptionRowTitle` so no worries on renaming the folder.